### ISSUE TITLE
causal joint model

### ISSUE THREAD

**Issue by amirabadiza921** [drizopoulos/JMbayes2#70]
Is it possible to analyze causal relationships such as mediator variables with this package?

**Comment by drizopoulos** [comment]
Yes, see here: [https://arxiv.org/abs/2309.02115](https://arxiv.org/abs/2309.02115).
